keda/templates/metrics-server/apiservice.yaml

```diff
@@ -3,11 +3,7 @@ kind: APIService
 metadata:
   annotations:
     {{- if .Values.certificates.certManager.enabled }}
-    {{- if .Values.certificates.certManager.generateCA }}
-    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Values.operator.name }}-ca
-    {{- else }}
-    cert-manager.io/inject-ca-from-secret: {{ .Release.Namespace }}/{{ .Values.certificates.certManager.caSecretName }}
-    {{- end }}
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Values.operator.name }}-tls-certificates
```

Comment:
was configured like its documented inside the cert-manager docs.
I'd suggest to follow 1:1 the docs of cert-manager:
https://cert-manager.io/docs/concepts/ca-injector/#injecting-ca-data-from-a-certificate-resource

Reply:
What do you mean?

Reply:
The annotation `cert-manager.io/inject-ca-from` has the value `namespace-name/certificate-name`

In the current implementation of the chart the value is `namespace-name/secret-name` which seems also working but it does not follow the documentation.

Reply:
you are right